src/ray/ray_config.h

```diff
@@ -109,7 +109,7 @@ class RayConfig {
         heartbeat_timeout_milliseconds_(100),
         num_heartbeats_timeout_(100),
         num_heartbeats_warning_(5),
-        initial_reconstruction_timeout_milliseconds_(200),
+        initial_reconstruction_timeout_milliseconds_(2000),
```

Comment:
Should this be set to something more conservative like 10000 until the bug is fixed?